packages/mobile/src/store/auth/reducer.ts

```diff
@@ -1,13 +1,19 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { DataStatus } from '~/common/enums/enums';
-import { signUp } from './actions';
+import { signUp, signIn } from './actions';
 
-type State = {
+export type State = {
   dataStatus: DataStatus;
+  accessToken: string;
+  refreshToken: string;
+  isLoggedIn: boolean;
```

Comment:
We don't need all of these. Let's put `user` here instead with default value of `null`.

For now you may return mocked empty object as a payload so that we can distinguish logged in user

Reply:
corrected